README.md

```diff
@@ -681,12 +681,13 @@ column `mytimestamp`:
         "type" : "jdbc",
         "jdbc" : {
 	        "url" : "jdbc:mysql://localhost:3306/test",
+            "statefile" : "statefile.json",
 	        "user" : "",
 	        "password" : "",
             "sql" : [
                 {
-                    "statement" : "select * from \"products\" where \"mytimestamp\" > ?",
-                    "parameter" : [ "$metrics.lastexecutionstart" ]
+                    "statement" : "select * from products where mytimestamp > ?",
+                    "parameter" : [ "$metrics.lastexecutionend" ]
```

Comment:
You are right!

2016-03-31 6:19 GMT+08:00 Sylwester Lachiewicz <notifications@github.com>:

> In README.md
> <https://github.com/jprante/elasticsearch-jdbc/pull/791#discussion_r57973813>
> :
>
> >  	        "user" : "",
> >  	        "password" : "",
> >              "sql" : [
> >                  {
> > -                    "statement" : "select * from \"products\" where \"mytimestamp\" > ?",
> > -                    "parameter" : [ "$metrics.lastexecutionstart" ]
> > +                    "statement" : "select * from products where mytimestamp > ?",
> > +                    "parameter" : [ "$metrics.lastexecutionend" ]
>
> Should be lastexecutionstart because in this case we will missed inserted
> data between 06:37:09.165 and 06:37:09.501
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jprante/elasticsearch-jdbc/pull/791/files/130f004ec2cbf99b32ddc8a18dd1a835adbcd954#r57973813>
>